db/migrate/20231210061330_add_created_at_to_fts_targets_with_index.rb

```diff
@@ -0,0 +1,106 @@
+# For auto load
+FullTextSearch::Migration
+
+class AddCreatedAtToFtsTargetsWithIndex < ActiveRecord::Migration[5.2]
+  def up
+    return if !table_exists?(:fts_targets)
+
+    add_column :fts_targets, :created_at, :timestamp
+
+    ActiveRecord::Base.transaction do
+      update_created_at_using_last_modified_at
+      update_created_at_using_created_on
+    end
+
+    if Redmine::Database.mysql?
+      add_index :fts_targets, :created_at
+    else
+      remove_index :fts_targets, name: "fts_targets_index_pgroonga"
+      add_index :fts_targets,
+                [:id,
+                 :source_id,
+                 :source_type_id,
+                 :project_id,
+                 :container_id,
+                 :container_type_id,
+                 :custom_field_id,
+                 :is_private,
+                 :last_modified_at,
+                 :created_at,
+                 :title,
+                 :content,
+                 :tag_ids],
+                using: "PGroonga",
+                with: "normalizer = 'NormalizerNFKC121'",
+                name: "fts_targets_index_pgroonga"
+    end
+  end
+
+  def down
+    return if !table_exists?(:fts_targets)
+
+    if Redmine::Database.mysql?
+      remove_index :fts_targets, :created_at
+    else
+      remove_index :fts_targets, name: "fts_targets_index_pgroonga"
+      add_index :fts_targets,
+                [:id,
+                 :source_id,
+                 :source_type_id,
+                 :project_id,
+                 :container_id,
+                 :container_type_id,
+                 :custom_field_id,
+                 :is_private,
+                 :last_modified_at,
+                 :title,
+                 :content,
+                 :tag_ids],
+                using: "PGroonga",
+                with: "normalizer = 'NormalizerNFKC121'",
+                name: "fts_targets_index_pgroonga"
+    end
+
+    remove_column :fts_targets, :created_at
+  end
+
+  private
+
+  class TmpTarget < ActiveRecord::Base
+    self.table_name = 'fts_targets'
+  end
+
+  class TmpType < ActiveRecord::Base
+    self.table_name = 'fts_types'
+  end
```

Comment:
How about using `FtsType`?

```suggestion
  class FtsType < ActiveRecord::Base
  end
```